demos/tensorflow-image-tagging/src/demoapp/utils.clj

```diff
@@ -0,0 +1,60 @@
+;;
+;;       Copyright (c) 2017 Two Sigma Investments, LP.
+;;       All Rights Reserved
+;;
+;;       THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF
+;;       Two Sigma Investments, LP.
+;;
+;;       The copyright notice above does not evidence any
+;;       actual or intended publication of such source code.
+;;
+(ns demoapp.utils
+  (:require [clj-time.core :as t]
+            [clj-time.format :as f]
+            [clojure.data.json :as json]
+            [clojure.string :as str])
+  (:import (java.util UUID)
+           java.util.regex.Pattern
+           org.joda.time.DateTime))
+
+(defn date-to-str [date-time & {:keys [format] :or {format "yyyy-MM-dd HH:mm:ss"}}]
+  (f/unparse
+    (f/with-zone (f/formatter format) (t/default-time-zone))
+    date-time))
+
+(defn stringify-elements
+  [k v]
+  (if (vector? v)
+    (map (partial stringify-elements k) v)
+    (cond
+      (instance? DateTime v) (date-to-str v)
+      (instance? UUID v) (str v)
+      (instance? Pattern v) (str v)
+      (= k :time) (str v)
+      :else v)))
+
+(defn- exception->strs
```

Comment:
This looks like a lot of copied code.   Did we really need these?  BTW I have a huge PR that renames/removes a lot of these functions.